packages/react-native-web/src/exports/Image/index.js

```diff
@@ -53,7 +53,14 @@ const resolveAssetUri = source => {
   if (typeof source === 'number') {
     // get the URI from the packager
     const asset = getAssetByID(source);
-    const scale = asset.scales[0];
+    let scale = asset.scales[0];
```

Comment:
we should cache the pixel ratio to prevent this from being called multiple times.

Reply:
`PixelRatio` (via `Dimensions`) does this already

Reply:
I mean to prevent this computation from running every time the asset is requested.